_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw_par.cpp

```diff
@@ -1961,16 +1963,8 @@ mfxStatus CheckVideoParam(MfxVideoParam& par, MFX_ENCODE_CAPS_HEVC const & caps,
     if (par.mfx.TargetUsage && caps.ddi_caps.TUSupport)
         changed += CheckTU(caps.ddi_caps.TUSupport, par.mfx.TargetUsage);
 
-    changed += CheckMax(par.mfx.GopRefDist, caps.ddi_caps.SliceIPOnly ? 1 : (par.mfx.GopPicSize ? std::max(1, par.mfx.GopPicSize - 1) : 0xFFFF));
+    changed += CheckMax(par.mfx.GopRefDist, (caps.ddi_caps.SliceIPOnly || IsOn(par.mfx.LowPower)) ? 1 : (par.mfx.GopPicSize ? std::max(1, par.mfx.GopPicSize - 1) : 0xFFFF));
 
-#if (MFX_VERSION >= MFX_VERSION_NEXT)
```

Comment:
Why is this part removed?
RAB are not supported on TGL tu7

Reply:
RAB-related lines cause #1805. Can't exactly say why.

Reply:
It looks like it was our guy who fixed sample_encode + mss recently. I will check with him

Reply:
Take a look at #1086 also. Looks like something missed here.